internal/index/indexoperations/index_test.go

```diff
@@ -64,3 +54,49 @@ func TestListIndexes(t *testing.T) {
 		t.Errorf("output does not match: %s", diff)
 	}
 }
+
+func TestAddIndexSuccess(t *testing.T) {
+	tmpDir, cleanup := testutil.NewTempDir(t)
+	defer cleanup()
+
+	indexName := "foo"
+	localRepo := tmpDir.Path("local/" + indexName)
+	initEmptyGitRepo(t, localRepo, "")
+
+	if err := AddIndex(tmpDir.Path("index"), indexName, localRepo); err != nil {
+		t.Errorf("error adding index: %v", err)
+	}
```

Comment:
IMHO, you should also test that `AddIndex` has actually done something.

Reply:
To do that, you can have a combined Add+List scenario.

Reply:
That way you can also move away from doing manual mkdir/clone in the test code.

Reply:
Sure I can change it to use that.